### ISSUE TITLE
How to select objects even there is 'noselectable' objects in front of it?

### ISSUE THREAD

**Issue by mudin** [fabricjs/fabric.js#5863]
Hello! I really enjoy using this fabric. Thanks a lot!
I've been trying to solve this by myself but took my weeks.
Still no luck.

The question:
I have not selectable object on front, and behind it there are selectable objects.
When I click the not selectable one, selectable ones should be clicked or selected.
and not selectable should be always in front.

here is the [jsfiddle](https://jsfiddle.net/mudin/70umgtja/)
Purple rectangle is not selectable, others are selectable.

Basically I have this project [indoorjs](https://mudin.github.io/indoorjs/) shows floorplan with markers and radar for indoor maps.
I cannot put radar in front since it blocks selecting and dragging markers.
If I find the solution, I will put it front.

If you guys help me or give a hint, it would be really appreciated!

Mudin

**Comment by mudin** [comment]
Sorry my bad. I didn't check the doc well. there is [evented](http://fabricjs.com/docs/fabric.Object.html#evented) property in object. it should set to `false`
